apps/emqx/src/emqx_cm.erl

```diff
@@ -375,34 +375,38 @@ discard_session(ClientId) when is_binary(ClientId) ->
 -spec kick_or_kill(kick | discard, module(), pid()) -> ok.
 kick_or_kill(Action, ConnMod, Pid) ->
     try
-        %% this is essentailly a gen_server:call implemented in emqx_connection
+        %% this is essentially a gen_server:call implemented in emqx_connection
         %% and emqx_ws_connection.
         %% the handle_call is implemented in emqx_channel
         ok = apply(ConnMod, call, [Pid, Action, ?T_KICK])
     catch
         _ : noproc -> % emqx_ws_connection: call
-            ok = ?tp(debug, "session_already_gone", #{pid => Pid, action => Action});
+            ?SLOG(debug, #{msg => "session_already_gone", pid => Pid, action => Action});
         _ : {noproc, _} -> % emqx_connection: gen_server:call
-            ok = ?tp(debug, "session_already_gone", #{pid => Pid, action => Action});
+            ?SLOG(debug, #{msg => "session_already_gone", pid => Pid, action => Action});
         _ : {shutdown, _} ->
-            ok = ?tp(debug, "session_already_shutdown", #{pid => Pid, action => Action});
+            ?SLOG(debug, #{msg => "session_already_shutdown", pid => Pid, action => Action});
         _ : {{shutdown, _}, _} ->
-            ok = ?tp(debug, "session_already_shutdown", #{pid => Pid, action => Action});
+            ?SLOG(debug, #{msg => "session_already_shutdown", pid => Pid, action => Action});
         _ : {timeout, {gen_server, call, _}} ->
```

Comment:
If I remember correctly, these `?tp` trace points are used in tests.

Reply:
eh, I will revert this.